plugin/pkg/admission/serviceaccount/admission.go

```diff
@@ -57,6 +57,8 @@ type serviceAccount struct {
 
 	// LimitSecretReferences rejects pods that reference secrets their service accounts do not reference
 	LimitSecretReferences bool
+	// RequireAPIToken determines whether pod creation attempts are rejected if no API token exists for the pod's service account
+	RequireAPIToken bool
```

Comment:
not sure... we don't have any config around admission plugins currently... this was more for testing and downstream consumption/customization